nav2_map_server/src/occ_grid_loader.cpp

```diff
@@ -223,7 +164,7 @@ nav2_util::CallbackReturn OccGridLoader::on_configure(const rclcpp_lifecycle::St
 
   // Create a service that loads the occupancy grid from a file
   load_map_service_ = node_->create_service<nav2_msgs::srv::LoadMap>(
```

Comment:
Shouldn't the load map service be in the map_server?

Reply:
This fulfilled in https://github.com/ros-planning/navigation2/commit/89ece9672dc847161607acc658b63fe2bafb90a7 when OccGridLoader was moved into MapServer